cmd/translatesfx/translatesfx/otel.go

```diff
@@ -284,11 +285,17 @@ func translateMonitors(sa saCfgInfo, cfg *otelCfg) (warnings []error) {
 		}
 	}
 
-	if _, ok := cfg.Receivers[processlist]; ok {
-		cfg.Service.Pipelines["logs"] = &rpe{
-			Receivers:  []string{processlist},
-			Processors: []string{resourceDetection},
-			Exporters:  []string{sfx},
+	for _, logReceiver := range []string{processlist, kubernetesEvents} {
+		if _, ok := cfg.Receivers[logReceiver]; ok {
+			if existingRPE, ok := cfg.Service.Pipelines["logs"]; ok {
+				existingRPE.Receivers = append(existingRPE.Receivers, logReceiver)
```

Comment:
You can make this `existingRPE.appendReceiver` so the receivers are sorted

Reply:
adopted this, but changed it a bit in the subsequent update to where isn't no longer needed.